skops/card/_model_card.py

```diff
@@ -0,0 +1,58 @@
+import re
+
+from modelcards import ModelCard
+from sklearn.utils import estimator_html_repr
+
+
+def _extract_estimator_config(model):
+    """Extracts estimator configuration and renders them into a vertical table.
+
+    Parameters
+    ----------
+        model (estimator): scikit-learn pipeline or model.
+
+    Returns
+    -------
+    str:
+        Markdown table of hyperparameters.
+    """
+    hyperparameter_dict = model.get_params(deep=True)
+    table = "| Hyperparameters | Value |\n| :-- | :-- |\n"
+    for hyperparameter, value in hyperparameter_dict.items():
+        table += f"| {hyperparameter} | {value} |\n"
+    return table
+
+
+def create_model_card(
+    model,
+    card_data,
+    **card_kwargs,
+):
+    """Creates a model card for the model and saves it to the target directory.
+
+    Parameters:
+    ----------
+    model: estimator
+        scikit-learn pipeline or model.
+    card_data: CardData
+        CardData object. See the
+        [docs](https://github.com/nateraw/modelcards/blob/main/modelcards/card_data.py#L78).
```

Comment:
IMHO, it's a bit dangerous to add a link to a specific line of code here. This information will become outdated easily when the file it points to is changed. Ideally, it should link to the documentation, but AFAICT, there is none.

I see two alternatives here:

1. Create a permalink. This can be achieved by visiting the github page and then pressing `y`. The URL bar will now show the [permalink](https://github.com/nateraw/modelcards/blob/7cb1c427a75c0a796631c137c5690ceedab3b2f8/modelcards/card_data.py#L78). This way, this line will always point to this specific version of the file, so there is no danger of it suddenly pointing to something else (or nothing). The disadvantage is that if the class is updated, this link does not reflect it.
2. It's a bit hacky, but you could also add a link to the search for "class CardData" like so: https://github.com/nateraw/modelcards/search?q=%22class+CardData%22. The advantage is that if the class is moved, the search will still find it. But if the class is renamed or if there are more classes with the same name, it won't work anymore.